.github/workflows/check-pr.yml

```diff
@@ -19,5 +20,16 @@ jobs:
       - name: Build
         run: make check-build
 
+  test-with-coverage:
+    if: github.event.pull_request.draft == false
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
       - name: Test
         run: make test
+        env:
+          CARGO_INCREMENTAL: '0'
+          RUSTFLAGS: '-Zprofile -Ccodegen-units=1 -Cinline-threshold=0 -Clink-dead-code -Coverflow-checks=off -Cpanic=abort -Zpanic_abort_tests'
+          RUSTDOCFLAGS: '-Zprofile -Ccodegen-units=1 -Cinline-threshold=0 -Clink-dead-code -Coverflow-checks=off -Cpanic=abort -Zpanic_abort_tests'
```

Comment:
The `-Clink-dead-code` flag is causing the CI build errors we're seeing, as it tries to link in all those WASM imports from `fvm_sdk::sys` that otherwise get ignored as dead code. Do we need that flag for the coverage reports?

Reply:
here's a small script to reproduce it locally
```bash
#!/bin/bash

export CARGO_INCREMENTAL=0
# breaks
#export RUSTFLAGS="-Zprofile -Ccodegen-units=1 -Cinline-threshold=0 -Clink-dead-code -Coverflow-checks=off -Cpanic=abort -Zpanic_abort_tests"
#export RUSTDOCFLAGS="-Zprofile -Ccodegen-units=1 -Cinline-threshold=0 -Clink-dead-code -Coverflow-checks=off -Cpanic=abort -Zpanic_abort_tests"
# works
export RUSTFLAGS="-Zprofile -Ccodegen-units=1 -Cinline-threshold=0 -Coverflow-checks=off -Cpanic=abort -Zpanic_abort_tests"
export RUSTDOCFLAGS="-Zprofile -Ccodegen-units=1 -Cinline-threshold=0 -Coverflow-checks=off -Cpanic=abort -Zpanic_abort_tests"
export RUSTC_BOOTSTRAP=1

cargo test --workspace
```